README.md

```diff
@@ -10,9 +10,7 @@ To that end, Brain-Score operationalizes experimental data into quantitative ben
 that any model candidate following the [`BrainModel`](brainscore_vision/model_interface.py) interface can be scored on.
 
 See the [Documentation](https://brain-score.readthedocs.io) for more details 
-and the [Tutorial](https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html) 
-and [Examples](https://github.com/brain-score/candidate_models/blob/master/examples/score-model.ipynb)
-for submitting a model to Brain-Score.
+and the [Tutorial](https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html) for submitting a model to Brain-Score.
```

Comment:
```suggestion
, e.g. for submitting a [model](https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html) or [benchmark](https://brain-score.readthedocs.io/en/latest/modules/benchmark_tutorial.html) to Brain-Score,
as well as the [Web Tutorial](https://www.brain-score.org/tutorial/), 
and [Examples](https://github.com/brain-score/vision/blob/master/examples).
```

Reply:
These links are included in the section right following this section. We can either keep these links in this section or the next section, but I recommend not both; having duplicated links in consecutive sessions could be confusing to users.

Reply:
it could be also hard to maintain since if the link is changed, multiple changes need to be made to keep them up to date

Reply:
you've got the better user perspective, I'll let you make the call!